app/views/categories/new.html.erb

```diff
@@ -0,0 +1,18 @@
+<%= render 'common/navigation', :title => 'Categories' %>
+<section class="access flex">
+  <%= form_with model: group, url: categories_url, method: :post do |f| %>
```

Comment:
Please add a "Back" button (<) on `Add a new category` page so when the user clicks it, it navigates the user to the home page.